static/locales/en-US.yaml

```diff
@@ -283,6 +283,7 @@ Settings:
     Fallback to Non-Preferred Backend on Failure: Fallback to Non-Preferred Backend
       on Failure
     Enable Search Suggestions: Enable Search Suggestions
+    Open Links In New Window: Open freetube:// Links in a New Window
```

Comment:
```suggestion
    Open Links In New Window: Open Deep Links Into FreeTube in a New Window
```

Reply:
or maybe 'URLs Passed to FreeTube" as this setting also applies to links passed via the command line

Reply:
Maybe we should just implement an tooltip message that explains what this does?

Reply:
Or just add tooltip giving examples of redirect extensions (mostly from those right?
Setting `External Link Handling` has one